src/config.rs

```diff
@@ -681,9 +686,19 @@ impl DbConfig {
                     );
                 })
         }
+
         if self.rate_bytes_per_sec.0 > 0 {
-            opts.set_ratelimiter(self.rate_bytes_per_sec.0 as i64);
+            if self.auto_tuned {
+                opts.set_ratelimiter_with_auto_tuned(
+                    self.rate_bytes_per_sec.0 as i64,
+                    self.rate_limiter_mod,
+                    true,
+                );
+            } else {
+                opts.set_ratelimiter(self.rate_bytes_per_sec.0 as i64);
```

Comment:
Do we need to handle `rate_limiter_mod` when the `auto_tuned` is false?

Reply:
yes, we need. From rocksdb's code, `set_ratelimiter` is a overload version of the function `set_ratelimiter_with_auto_tuned` with default parameter "mode=WritesOnly, auto_tuned=false".   So I will remove the `else` branch, and use a unified function call to cover those cases.